static/css/library/status.css

```diff
@@ -192,17 +191,31 @@ div.modal-body div#movie_settings{
     margin: 0 0 1em 0;
 }
 
+div.modal-body div#movie_settings select{
+  display: block;
+}
+
+div .modal-body div#movie_settings div{
+  float: left;
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+}
+
+div.modal-body div#movie_settings a#update_movie_options {
+  margin-top: 20px;
+}
+
 div.modal-body p.plot{
     text-indent: 2em;
-    margin: 1em;
 }
 
     /* Search Results table */
 div.modal-body div#search_results_table{
     overflow-y: scroll;
     max-height: 17em;
-    margin: 1em 0 0 0;
     transition: none;
+    padding-left: 0px;
+    padding-right: 0px;
```

Comment:
I like the padding because the search results box edges line up with the poster and plot box. What do you think about leaving this padding, but setting `div.modal-body { padding: 1.25em 0}` to slim things down a little. We don't lose the visual alignment but it saves a bunch of space.